src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/commands/set_interior_vehicle_data_request_test.cc

```diff
@@ -155,55 +157,58 @@ class SetInteriorVehicleDataRequestTest
 
 TEST_F(SetInteriorVehicleDataRequestTest,
        Execute_ValidWithoutReadOnlyParams_ExpectResendToHMI) {
+  using namespace rc_rpc_plugin::message_params;
+
   // Arrange
-  MessageSharedPtr mobile_message = CreateBasicMessage();
-  ns_smart_device_link::ns_smart_objects::SmartObject& msg_params =
+  auto mobile_message = CreateBasicMessage();
+  auto& msg_params =
       (*mobile_message)[application_manager::strings::msg_params];
-  msg_params[message_params::kModuleData][message_params::kModuleType] =
-      mobile_apis::ModuleType::CLIMATE;
-  smart_objects::SmartObject climate_control_data(smart_objects::SmartType_Map);
-  climate_control_data[message_params::kFanSpeed] = 10;
+  msg_params[kModuleData][kModuleType] = mobile_apis::ModuleType::CLIMATE;
+  auto& control_data = msg_params[kModuleData][kClimateControlData];
+  const uint64_t fan_speed = 10;
```

Comment:
@mvorobio Don't use implicit conversions. Please add postfix `u` to the number.

Reply:
@AByzhynar please check https://github.com/mvorobio/sdl_core/pull/2/commits/30f301a618a68f379d67d62adb12b354a002337f
I'd prefer `const auto fan_speed = static_cast<uint64_t>(10);`